apps/docs/src/development/theming.md

```diff
@@ -0,0 +1,20 @@
+# Theming Setup
+
+Onyx supports a dark and a light theme.  
+To learn more about the theming concept of onyx, take a look at our [colors documentation](/basics/colors.html)
+
+In order to let the user choose the preferred theme, we recommend [useDark of vueuse](https://vueuse.org/core/useDark/#usedark).  
+Combine it as follows with an onyx button to let the users set the theme of their choice:
+
+```vue
+<script lang="ts" setup>
+import { useDark, useToggle } from "@vueuse/core";
+import { OnyxButton } from "sit-onyx";
+
+const toggleDark = useToggle(useDark());
```

Comment:
Usually you also need a ref whether its currently dark or not because Jannick intended to show the current theme in the nav bar user menu